src/main/kotlin/me/kject/internal/Registry.kt

```diff
@@ -1,22 +1,21 @@
 package me.kject.internal
 
-import me.kject.annotation.Facade
-import me.kject.annotation.Require
-import me.kject.annotation.UseConstructor
+import me.kject.annotation.*
 import me.kject.dependency.trace.DependencyTraceBuilder
 import me.kject.dependency.trace.RequestType
-import me.kject.exception.create.CircularDependencyException
-import me.kject.exception.create.IllegalFacadeException
-import me.kject.exception.create.MultipleConstructorsException
-import me.kject.exception.create.NoConstructorException
+import me.kject.exception.DisposeFailedException
+import me.kject.exception.call.CallFailedException
+import me.kject.exception.create.*
 import me.kject.internal.call.Caller
-import java.util.Collections
+import java.util.*
 import kotlin.reflect.KClass
 import kotlin.reflect.KFunction
 import kotlin.reflect.full.*
 
 internal object Registry {
 
+    var allowCreate = false
```

Comment:
does this need to be public?

Reply:
As it is in an internal object, it's acutally internal.

It needs to be internal, as the `KJectImpl` sets it twice (see line 52 and 61).
If you have a better idea on how to manage that, i'm open to discuss those.

Reply:
Oh, that's fully fine. I didn't notice that the whole `Registry` was internal anyway. Originally, my idea was to make the field `internal`, but in that case, that's redundant.